### PR TITLE
New 'Staking' and small other changes

### PR DESCRIPTION
**Pull request by BreadJS**
## Abstract
I've changed the design of the 'staking' page to make it more responsive and have a cleaner UI.

## What does this PR address?
MPW was not really responsive for other devices and lacks a good design (IMO). My intend is to redesign MPW for a better look and also make it as responsive as possible for every device.

## What features or improvements were added?
1. New `Staking` page
    - Used the same transfer menu to `Stake` and `Unstake`
      - Amount to `Stake` and `Unstake` is now on the Amount section
    - New `Activity` feed and `Load more` button
      - `Time` column will show tx time in last 24 and date if tx is older 24 hours
      - `Time` will be shown in 12 hour format (AM/PM)
      - Has a confirmation checkmark
    - Activity text also shows amount of PIV
2. Other changes
    - Replaced all `X` buttons on modals with a better visible one
    - Changed `pointer: cursor;` location in dashboard dropdown

**Note:** *JSKitty will complete the hash shortening function. Currently it only shows the first 24 characters of the hash*

## How does this benefit users?
A better design will lure more users in using MPW. This also applies for making the site more responsive.

## PIVX Address
`DQnmBB6b6xLdjt2kqfgDWJGZGoMSATp1xS`

### CODE REVIEW

**Comment by Duddino** [comment]
>     2. Other changes
>        
>        * Replaced all `X` buttons on modals with a better visible one
>        * Changed `pointer: cursor;` location in dashboard dropdown
Could you put this in a different PR? It makes reviewing easier and faster.

I really like the rewards table; Not really sure about the stake box, it's similar enough to the dashboard one, maybe the two can somehow be merged?

**Comment by Duddino** [comment]
image

File: scripts/global.js [hunk 1]
```diff
@@ -475,25 +463,55 @@ export async function updateStakingRewardsGUI() {
...
+            <tr>
+                <th scope="col" class="tx1">Time</th>
+                <th scope="col" class="tx2">Hash</th>
+                <th scope="col" class="tx3">Amount</th>
+                <th scope="col" class="tx4 text-right"></th>
+            </tr>
+        </thead>
+        <tbody>`;
+
+        let i = true;
```
Comment (Duddino) [inline scripts/global.js:478]:
```suggestion
```

File: scripts/global.js [hunk 2]
```diff
@@ -475,25 +463,55 @@ export async function updateStakingRewardsGUI() {
...
+    arrRewards.map(
```
Comment (Duddino) [inline scripts/global.js:479]:
```suggestion
    arrRewards.forEach(
```

File: scripts/global.js [hunk 3]
```diff
@@ -475,25 +463,55 @@ export async function updateStakingRewardsGUI() {
...
+        (cReward) => {
+            let dateTime = new Date( cReward.time * 1000 );
```
Comment (Duddino) [inline scripts/global.js:481]: Could you refactor the date algorithm to a new function?

**Review by Duddino — CHANGES_REQUESTED** [review]

File: scripts/global.js (hunk 3 above)
Comment (JSKitty) [reply scripts/global.js]: I've done some "trimming" and it's quite small now, do you still think it should be shipped to a new function? I think it's now not worth it due to the tiny size.

Comment (Duddino) [reply scripts/global.js]: I think it's long enough, plus it will be reused in other places

**Review by Duddino — COMMENTED** [review]
I like how the PR turned out, just two nits on my part

**Review by Duddino — APPROVED** [review]
tACK, staking/unstaking works and rewards display correctly

**Review by JSKitty — APPROVED** [review]
**tACK** 0988aa4b3d5525253c705c28e61451b7ab491703, `Stake` and `Unstake` works well; responsive at all of my tested screen sizes, refresh works nicely and is equally responsive.

LGTM.

**Review by Duddino — APPROVED** [review]
tACK looks good and works let's get this merged

**Review by Liquid369 — APPROVED** [review]
tACK 37c26c55ee70163e6a9f0c82c391ab67a27a6309
Looks good on both desktop and mobile
Staking/Unstaking is working fine as well

**Review by JSKitty — APPROVED** [review]
**re-tACK** 37c26c55ee70163e6a9f0c82c391ab67a27a6309

**Pull request merged**